service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/EncounterMapper.java

```diff
@@ -128,4 +141,20 @@ private void setServiceProvider(Encounter encounter, POCDMT000002UK01ClinicalDoc
         encounter.setServiceProvider(serviceProvider);
         encounter.setServiceProviderTarget(serviceProviderOrganization);
     }
+
+    private void setSubject(Encounter encounter, POCDMT000002UK01ClinicalDocument1 clinicalDocument1) {
+        if (clinicalDocument1.sizeOfRecordTargetArray() == 1) {
+            POCDMT000002UK01PatientRole patientRole = clinicalDocument1.getRecordTargetArray(0).getPatientRole();
```

Comment:
If there will be no patienRole set, I think null pointer will be thrown in the PatientMapper. Please make sure and if so, add proper condition.

Reply:
Modified the module to prevent null pointer exception.